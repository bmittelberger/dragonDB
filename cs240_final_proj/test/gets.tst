open dragon_store
get armor
get space
get origami
get magic_dragon
get dragon_ball_z
