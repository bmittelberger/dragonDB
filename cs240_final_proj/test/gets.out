armor : 8
space : 10
origami : 2
magic_dragon : 9
dragon_ball_z : 10